### ISSUE TITLE
Blog formatting when adding white box style

### ISSUE THREAD

**Issue by russ-manta** [MantaRayMedia/alnap#286]
### Done criteria
- [ ] 'Tried to frame a para in a blog in white box format and, although the code in the backend looks right, the box is framing the whole text in that nugget, not the bit I selected...' http://alnap-dev.mrmdev.co.uk/blogs/responding-to-changing-needs-can-the-humanitarian-sector-adapt-to-the-new-normal

---
### Designs
- [ ] ?
- [ ] ?

---
### ALNAP to check
- [ ] ?
- [ ] ?

**Comment by nadia847** [comment]
@Maria-alnap The white box style is designed to frame the entire nugget. If you would like to just frame a bit of text in there, you should separate into different nuggets.

**Comment by Maria-alnap** [comment]
@nadia847 Aaaah! I thought that would be the case. Great, thanks for clarifying. will move to done

**Comment by nadia847** [comment]
@Maria-alnap yeah it was basically built just to do the white box on the contact us page :o)

**Comment by nadia847** [comment]
@Maria-alnap oh and I've reduced spacing between nuggets in the main content region so it should be easier to do this without content looking tot spaced out.
